api/services/control/control.proto

```diff
@@ -36,6 +36,7 @@ message PruneRequest {
 
 message DiskUsageRequest {
 	repeated string filter = 1; 
+	int64 ageLimit = 2;
```

Comment:
Struggled to find a good name for this. In Prune it is called `KeepDuration` but that name does not make sense at all for DiskUsage.

In buildx this is `--filter until=<duration>` that is detected on client side and converted to another field. `buildx du` will likely keep the same name there.

Reply:
`ageLimit` SGTM - but if we change this, would it also make sense to change the name in the prune request? We're already making changes to these structs in the next release because of the new storage fields.

Reply:
if prune had `ageLimit` is it obvious which way it works? Eg. is it pruning the things that are newer than `ageLimit` or older? Correct is older but I don't know if it is obvious.

Reply:
If anything, I think this is `minAge`